cwatch.php

```diff
@@ -422,10 +463,53 @@ public function cancelService($package, $service, $parent_package = null, $paren
     public function getClientTabs($package) 
     {
         return [
-            'tabClientActions' => Language::_('Cwatch.tab_client_actions', true)
+            'tabClientActions' => Language::_('Cwatch.tab_client_actions', true),
+            'tabClientMalWare' => Language::_('Cwatch.site.malware', true)
         ];
     }
-
+    function tabClientMalWare($package, $service, array $get = null, array $post = null, array $files = null) 
+    {
+        $this->view = new View('tab_malware', 'default');
+        $this->view->base_uri = $this->base_uri;
+        // Load the helpers required for this view
+        Loader::loadHelpers($this, array('Form', 'Html'));
+        $row = $this->getModuleRow();
+        $service_fields = $this->serviceFieldsToObject($service->fields);
+        $licensekey = $service_fields->licensekey;
+        if (isset($row->meta->username)) {
+            $user = $row->meta->username;
```

Comment:
Line 497 loads the API by passing in the `$user`, `$pass`, and `$sandbox`, but those variables are only conditionally defined. If any are not set, this will cause an error.